nwbib/nwbib.ttl

```diff
@@ -371,7 +403,7 @@
     skos:notation "141460" ;
     skos:prefLabel "Mineralquellen. Thermalquellen"@de ;
     skos:scopeNote "Freie SW: Mineralquelle bzw. Thermalquelle"@de ;
-    skos:exactMatch wd:Q1365924, wd:Q177380, gnd:4170037-5, gnd:4191517-3 .
+    skos:exactMatch gnd:4170037-5, gnd:4191517-3, wd:Q1365924, wd:Q177380 .
```

Comment:
Das ist doch ein klarer Fall für `skos:narrowMatch`, oder?

Reply:
Hallo Adrian, ich verstehe noch nicht, warum du skos:narrowMatch vorschlägst. Kannst du mir das nochmal erklären? Zum Beispiel bei 141460 Mineralquelle. Thermalquelle. Ich habe mir nochmal die Hilfen für SKOS-Elemente angeschaut. Skos:narrowMatch wird verwendet, wenn es sich um einen Unterbegriff handelt. Da, wo die NWBib-Sachsystematik zwei Begriffe hinter einer Notationszahl (hier 141460) hat, sind die beiden Begriffe gleichwertig. 
Oder beziehst du dich auf die geplante Darstellung von GND- & Wikidata-Links hinter den einzelnen Notationen, die wir für die Sachsystematik geplant haben? Wenn man zwei GND-Links einbinden möchte, sieht das wahrscheinlich nicht gut aus...Oder können wir es doch so machen, dass sich beim Klick auf eine Notation ein Feld unterhalb der Notation öffnet, worin die gesetzten Links dargestellt werden? Vielleicht sollten wir beide uns hierzu noch per Videokonferenz nach deinem Urlaub austauschen. Dann kann ich den PR fertigstellen.

Reply:
Hier mein Gedankengang. 

- 141460  Mineralquellen. Thermalquellen: Ich gehe davon aus, dass diese Systemsstell sich auf Dinge bezieht, die entweder unter den Begriff  "Mineralquelle" oder den Begriff "Thermalquelle" oder unter beide Begriffe fallen. Ferner gehe ich davon aus, dass diese beiden Begriffe nicht identisch sind und dass auch nicht ein Begriff ein Überbegriff des anderen ist. Ich gehe davon aus, dass sie sich in ihrer exensionalen Bedeetung allenfalls überschneiden, d.h. einige Thermalquellen sind auch Mineralquellen aber nicht alle.
- https://d-nb.info/gnd/4170037-5 & http://www.wikidata.org/entity/Q1365924: Diese URIs stehen für den Begriff "Mineralquelle".
- https://d-nb.info/gnd/4191517-3 & http://www.wikidata.org/entity/Q177380: Diese URIs stehen für den Begriff "Thermalquelle".
- D.h., der Begriff "Mineralquellen. Thermalquellen" umfasst sowohl "Mineralquelle" und "Thermalquelle", ist aber nciht identisch mit einem, sondern nur mit der Vereinigung der beiden Begriffe. Das heißt, dass die Einzelbegriffe "Mineralquelle" und "Thermalquelle" Unterbegriffe von "Mineralquellen. Thermalquellen" sind.
- Hier passt dann `skos:narrowMatch` das eben zu Unterbegriffen in einem anderen Vokabular verlinkt.

Das ist analog dazu, wenn der STW von Kapitalismus mit `skos:narrowerMatch` auf Spätkapitalismus in der GND verlinkt: http://zbw.eu/stw/descriptor/11046-0

Reply:
Danke für die Erklärung. Dann setz ich das mal um. 
Eine Frage noch: Mit "geschlechtsunspezifische Berufsbezeichnung" meinst du z.B. "Journalisten", "Verleger" (Plural)?

Reply:
> Eine Frage noch: Mit "geschlechtsunspezifische Berufsbezeichnung" meinst du z.B. "Journalisten", "Verleger" (Plural)?

Ja, wenn das generische Maskulinum verwendet wird oder sowas wir "Journalist:in", "Verleger\*in" etc.